src/index.js

```diff
@@ -1,17 +1,7 @@
-import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from "./App";
```

Comment:
Would be better also to use TypeScript. But as far as I remember it's not mandatory but improve your skill as well. Currently most of the React project setting up with TS. As you wish :)